### PR TITLE
Make messages with safe(r) transport.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM python:3.4-wheezy
 RUN apt-get update &&\
-    apt-get install -y --no-install-recommends libpq-dev gettext libjpeg8-dev postgresql-client &&\
+    apt-get install -y --no-install-recommends libpq-dev gettext libjpeg8-dev postgresql-client sharutils &&\
```
Comment (glogiotatidis) [inline Dockerfile]: We need that fo uuencode. I also update jenkins ansible script https://github.com/glogiotatidis/jenkins-ansible-playbook/commit/921f8b775b4343762af1ecbb68881016a4e55e68

File: bin/jenkins/makemessages.sh [hunk 2]
```diff
@@ -18,12 +18,12 @@ virtualenv $TDIR
...
 
-rm -rf locale db-strings.txt
+rm -rf locale db-strings.txt run-output
 git clone $LOCALE_REPOSITORY locale
 
 deis login $DEIS_CONTROLLER --username $DEIS_USERNAME --password $DEIS_PASSWORD
-deis run -a $DEIS_APP -- "./manage.py runscript db_strings && cat db-strings.txt" > db-strings.txt
-
+deis run -a $DEIS_APP -- "./manage.py runscript db_strings && echo CUTHERE && uuencode db-strings.txt -" > run-output
+awk '{if (nowprint) {print;}}/CUTHERE/ {nowprint = 1}' run-output | uudecode > db-strings.txt
```
Comment (glogiotatidis) [inline bin/jenkins/makemessages.sh]: `deis run` and `runscript db_strings` generate output that we don't want. We use `CUTHERE` and `awk` scripting to separate this output from `uuencode` output

**Comment by glogiotatidis** [comment]
@jgmize r?

File: bin/jenkins/makemessages.sh (hunk 2 above)
Comment (jgmize) [inline bin/jenkins/makemessages.sh]: Nicely done.

**Pull request merged**